doc/conf.py

```diff
@@ -61,6 +61,7 @@
     # will work without this - for canonical (so can ignore when building locally or test deploying)
     'WEBSITE_SERVER': 'https://hvplot.pyviz.org',
     'VERSION': version,
+    'GOOGLE_ANALYTICS_UA': 'UA-154795830-1',
```

Comment:
```suggestion
    'GOOGLE_ANALYTICS_UA': 'UA-154795830-5',
```

Reply:
I pulled that key from I believe Datashader, should I be incrementing that last number for each project I add it to?